GOVERNANCE.md

```diff
@@ -1,88 +1,234 @@
+
+## Core Maintainer
+
+Core Maintainers are responsible for the Backstage project as a whole. They help review and merge project-level pull requests as well as coordinate work affecting multiple project areas. A core maintainer needs to commit the majority of their working time towards the project, and exercise judgment for the good of the project, independent of their employer. Core maintainers should also mentor and seek out new maintainers, lead community meetings, and communicate with the CNCF on behalf of the project. To become a core maintainer one needs to have been the maintainer of a number of different project areas, demonstrate a deep knowledge of large parts of the Backstage project, and be backed by the existing core maintainers.
+
+A Core Maintainer have all the rights and responsibilities of a Project Area Maintainer.
+
+### Responsibilities
+
+- Take part in the incoming issue and PR triage and review process. PRs are shared equally among all maintainers
+- Mentor new Project Area Maintainers and Plugin Maintainers
+- Drive refactoring and manage tech health across the entire project
+- Participate in CNCF maintainer activities
+- Respond to security incidents in accordance to our [security policy](./SECURITY.md)
+- Determine strategy and policy for the project
+- Participate in or leading community meetings
+
+### Requirements
+
+- Experience as a Project Area Maintainer for at least 6 months
+- Demonstrates a broad knowledge of the project across multiple areas
+- Is able to exercise judgment for the good of the project, independent of their employer, friends, or team
+- Mentors other contributors
+- Can commit to spending at least 10 days per month working on the project
+
+### Privileges
+
+- Approve PRs that fall outside any specific project area
+- Merge PRs to any area of the project
+- Represent the project in public as a Maintainer
+- Communicate with the CNCF on behalf of the project
+- Have a vote in Maintainer decision-making meetings
+
+### Becoming a Core Maintainer
+
+Any core maintainer or end user sponsor may nominate a new core maintainer by opening a PR towards the [OWNERS.md](OWNERS.md) file. Core maintainers must be approved by a majority of the existing core maintainers and end user sponsors.
+
+## End User Sponsors
+
+### Role of a Backstage End User Sponsor
+
+- Provide support for Backstage by removing blockers, securing funding, providing advocacy, feedback, and ensuring project continuity and long term success
+- Assist Backstage maintainers in prioritizing upcoming roadmap items and planned work
+- Provide neutral mediation for any disputes that arise as part of the project
+
+### Backstage End User Sponsor Membership
 
 The End User Sponsors group comprises at most 5 people. To be eligible for membership in the group, you or the company where you work you must:
```

Comment:
There's no limit placed here on the number of employees of one company who can be End User Sponsors, or any criteria based on which a specific company would have more than one EUS rep.  Together with the weak requirements for qualifications, under these rules a single company to take and keep 100% all EUS seats feels almost inevitable.

I strongly recommend that you instead either have a one-seat-per-company rule, or some formula for multiple reps which does not result in one company having a majority of EUS seats (at least, after a suitable bootstrap period).

Reply:
Yep makes sense, but I'll want to leave this for a separate review of the governance structure when solidly out of the bootstrap period.